types/pulltorefreshjs/index.d.ts

```diff
@@ -3,9 +3,14 @@
 // Definitions by: Daniel Rosenwasser <https://github.com/DanielRosenwasser>
 //                 Tamas Kinsztler <https://github.com/humpedli>
 //                 Samuel Bodin <https://github.com/bodinsamuel>
+//                 Stefan Langeder <https://github.com/slangeder>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-export function init(options?: Options): void;
+export function init(options?: Options): PullToRefreshInstance;
+
+export interface PullToRefreshInstance {
+	destroy(): void;
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/pulltorefreshjs/index.d.ts;
  Line: 12;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.